recipes/foamalgo/install_python.sh

```diff
@@ -0,0 +1,2 @@
+git submodule update --init
```

Comment:
we don't want to vendor the dependencies, that is the main reason we are against it.
You should have a way to build this package without vendoring dependencies. Otherwise you will need to patch this package to build it properly

Reply:
OK! Finally, I removed the sub-modules in my packages. So it does not vend dependencies. But the CI has different errors that I cannot understand. I am bit frustrated but will continue tomorrow. Thanks again for all the comments.